web/src/pages/home/homePage.js

```diff
@@ -674,7 +643,7 @@ const Homepage = () => {
         <div className="groups-container">
           {groups.map((group, index) => (
             <div className="group-card" key={index}>
-              <a href="https://www.domits.com/home/" target="_blank" rel="noopener noreferrer">
+              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```

Comment:
Dit moet global zijn, niet specifiek naar domits.com
```suggestion
              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```